asm/compiler.asm

```diff
@@ -136,17 +136,20 @@ COLON
     jmp RBRAC ; enter compile mode
 
 
-; --- HEADER ( name -- )
+.empty_parse
+    jmp print_word_not_found_error
```

Comment:
What will be printed here? I guess a `?`? In that case maybe the `err` printout of .on_stack_underflow would be a better jump target.